sources/packages/web/src/services/http/dto/Student.dto.ts

```diff
@@ -34,10 +34,7 @@ export interface StudentProfileAPIOutDTO {
   pdStatus: StudentPDStatus;
   sin: string;
   sinConsent: boolean;
-}
-
-export interface AESTStudentProfileAPIOutDTO extends StudentProfileAPIOutDTO {
-  hasRestriction: boolean;
+  hasRestriction?: boolean;
```

Comment:
why is this change? `hasRestriction` is only needed by the ministry right?

Reply:
we can use 2 DTO in the client methods, I believe we did it in many other shared API client methods right? Let me know if I am missing anything

Reply:
Currently most of the DTOs are not client specific and I was trying to follow that pattern here.

Reply:
As discussed,  we can discuss  it in dev syncup